extensions/wikia/ContentReview/ContentReviewHelper.php

```diff
@@ -90,4 +92,17 @@ public static function isContentReviewTestModeEnabled() {
 
 		return $contentReviewTestModeEnabled;
 	}
+
+	public function isPageInReviewProcess() {
```

Comment:
I think this method name is a little bit misleading, because it checks is a current page in review process only on diff pages and you cannot use it somewhere else.